tests/integration/cache_test.go

```diff
@@ -1,18 +1,4 @@
-// Copyright 2025 The etcd Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package cache
+package integration
```

Comment:
In integration tests we use different package to setup etcd. 

Please remove functions for setting up etcd and for each tests use the following code:
```
import "go.etcd.io/etcd/tests/v3/framework/integration"

func TestX(*testing.T) {
	integration.BeforeTest(t)

	clus := integration.NewCluster(t, &integration2.ClusterConfig{Size: 1})
	defer clus.Terminate(t)

	cc, err := clus.ClusterClient(t)
	require.NoError(t, err)
	...
}
```